app/v1/asset/msgs.go

```diff
@@ -24,10 +24,11 @@ var (
 	MaximumAssetSymbolMinAliasSize = 10                    // maximal limitation for the length of the asset's symbol_min_alias
 	MaximumAssetNameSize           = 32                    // maximal limitation for the length of the asset's name
 
-	MinimumGatewayMonikerSize = 3   // minimal limitation for the length of the gateway's moniker
-	MaximumGatewayMonikerSize = 8   // maximal limitation for the length of the gateway's moniker
-	MaximumGatewayDetailsSize = 280 // maximal limitation for the length of the gateway's details
-	MaximumGatewayWebsiteSize = 128 // maximal limitation for the length of the gateway's website
+	MinimumGatewayMonikerSize  = 3    // minimal limitation for the length of the gateway's moniker
+	MaximumGatewayMonikerSize  = 8    // maximal limitation for the length of the gateway's moniker
+	MaximumGatewayIdentitySize = 3000 // maximal limitation for the length of the gateway's identity
```

Comment:
3000 is too large for identity, how about 128?

Reply:
Done